programs/bpf_loader/src/lib.rs

```diff
@@ -1096,6 +1096,23 @@ fn process_loader_upgradeable_instruction(
                         ic_logger_msg!(log_collector, "Program account not owned by loader");
                         return Err(InstructionError::IncorrectProgramId);
                     }
+                    if invoke_context
```

Comment:
So post feature activation the clock will always be required to close the account?  Are the tools going to also be updated?  No reason to not update the tools immediately to pass the clock sysvar

Reply:
The PR also adjusts the `bpf_loader_upgradeable::close_any()` factory method and the parser for the explorer. Is there any other place affected?

Also, I am not sure if explicitly passing the sysvar is strictly necessary anymore. We could potentially just load the clock from the sysvar cache without it being passed as an account, right?

Reply:
looks like we did switch to sysvar cache for the `Clock`. this conversation should be resolved?